frontend/src/containers/__tests__/TopicareaSettings.test.tsx

```diff
@@ -9,7 +9,18 @@ test("renders the title", async () => {
   const { getByRole } = render(<TopicareaSettings />, {
     wrapper: MemoryRouter,
   });
-  expect(getByRole("heading", { name: "Topic areas" })).toBeInTheDocument();
+  expect(
+    getByRole("heading", { name: "Topic areas (Ministries)" })
+  ).toBeInTheDocument();
+});
+
+test("renders the title with set value of topic areas", async () => {
+  const { getByRole } = render(<TopicareaSettings />, {
+    wrapper: MemoryRouter,
+  });
+  expect(
+    getByRole("heading", { name: "Topic areas (Ministries)" })
```

Comment:
As we discussed in the standup we need to use here the topicAreaLabel we just set and not longer use the ()

Reply:
That's true. I guess you no longer need the `determineTopicAreaString` function.

Reply:
Done!